src/utils/flags/flagtypes.h

```diff
@@ -154,21 +169,25 @@ Value WriteFixedValueType ( T& target, const T &value ) {
 /**
  * Returns a \sa ZyppFlags::Value instance handling flags taking a string parameter representing a \sa zypp::filesystem::PathName
  */
-Value PathNameType( filesystem::Pathname &target, const boost::optional<std::string> &defValue, std::string hint );
+Value PathNameType( filesystem::Pathname &target, const boost::optional<std::string> &defValue = boost::optional<std::string>(), std::string hint = std::string() );
```

Comment:
JFYI: it's just `(zypp::)Pathname`. No one calls it   `(zypp::)filesystem::Pathname`.